examples/rxjs/src/index.ts

```diff
@@ -1,7 +1,9 @@
 import { Stream } from 'xstream';
-import { Observable } from 'rxjs';
+import { Observable, of as observableOf, pipe } from 'rxjs';
```

Comment:
The `pipe` is not used here.
```suggestion
import { Observable, of as observableOf } from 'rxjs';
```